py3status/modules/sysdata.py

```diff
@@ -326,6 +348,14 @@ def sysData(self):
             self.values['swap_unit'] = swap_unit
             self.py3.threshold_get_color(swap_used_percent, 'swap')
 
+        # get load average
+        if self.py3.format_contains(self.format, 'load*'):
+            load1, load5, load15 = self.data.load()
+            self.values['load1'] = load1
+            self.values['load5'] = load5
+            self.values['load15'] = load15
+            self.py3.threshold_get_color(load1, 'load')
```

Comment:
we sort of do this with cpu because that's a percent as well.  It's just that otherwise if I have thresholds then they trigger for everything but load.  I'll have a think

@ultrabug you have a view here?